services/filemanager/s3manager.go

```diff
@@ -155,12 +167,16 @@ func GetS3Config(config map[string]interface{}) *S3Config {
 	if config["accessKey"] != nil {
 		accessKey = config["accessKey"].(string)
 	}
-	return &S3Config{Bucket: bucketName, Prefix: prefix, AccessKeyID: accessKeyID, AccessKey: accessKey}
+	if config["enableSSE"] != nil {
+		enableSSE = config["enableSSE"].(bool)
```

Comment:
What happens if that is a string instead? Does it default to false or will it crash?